src/borg/archiver.py

```diff
@@ -862,14 +862,14 @@ def _list_archive(self, args, repository, manifest, key, write):
 
     def _list_archives(self, args, repository, manifest, key, write):
         archives = self._get_archives_slice(args, manifest)
-        for i, archive in enumerate(archives):
-            write('Contents of {} ({}/{}):'.format(archive.name, i+1, len(archives)))
+        for i, archive in enumerate(archives, 1):
+            write('Contents of {} ({}/{}):'.format(archive.name, i, len(archives)))
             args.location.archive = archive.name
             self._list_archive(args, repository, manifest, key, write)
             if self.exit_code:
                 break
             if len(archives) - i > 1:
-                write('\n')
+                write()
```

Comment:
ehrm, guess you need to keep the \n here. write() does not write one by default, like print() does.